docs/release-notes.md

```diff
@@ -13,9 +25,9 @@ Major changes:
 
 Minor changes:
 
+- providers: Add Hetzner Cloud
```

Comment:
Hmm, this should be in the "Major changes" section I think. It's on equal footing with the Scaleway change. And maybe let's reword it so it's similar to that one. (I.e. "Add support for Hetzner Cloud".)

Reply:
Thanks! Updated